lib/tldr-lint-cli.js

```diff
@@ -115,6 +111,9 @@ if (require.main === module) {
     .option('-v, --verbose', 'print verbose output')
     .parse(process.argv);
 
-  cli.process(args);
+  if (program.args.length !== 1) {
+    program.help();
```

Comment:
https://github.com/tj/commander.js#display-help-from-code
```suggestion
    program.help({ error: true });
```